tf2/include/tf2/convert.h

```diff
@@ -72,6 +72,13 @@ tf2::TimePoint getTimestamp(const T & t);
 template<class T>
 std::string getFrameId(const T & t);
 
+/**\brief Get the covariance matrix from data
+ * \param[in] t The data input.
+ * \return The covariance matrix associated with the data.
+ */
+template<class T>
+std::array<std::array<double, 6>, 6> getCovarianceMatrix(const T & t);
```

Comment:
Please add `#include <array>` at the top of this file for this.

Reply:
Fixed in 34246f8.